icons/shredder.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Alirashidy",
```

Comment:
Should also include every contributor from file.
Now that I think about it we could think about naming this something like `file-shredder` as well 🤔